PWG/EMCAL/EMCALtasks/AliEmcalCorrectionCellTimeCalib.cxx

```diff
@@ -57,6 +59,11 @@ Bool_t AliEmcalCorrectionCellTimeCalib::Initialize()
   if (!fRecoUtils)
     fRecoUtils  = new AliEMCALRecoUtils;
 
+  bool doCalibTimeEdep = kFALSE;
+  GetProperty("doCalibTimeEdep", doCalibTimeEdep);
+  fCalibrateTimeVsE = doCalibTimeEdep;
```

Comment:
You can set `fCalibrateTimeVsE` directly with `GetProperty("doCalibTimeEdep", fCalibrateTimeVsE);` (unless I miss a negation somewhere or something)

Reply:
fixed